### ISSUE TITLE
Make this available to composer

### ISSUE THREAD

**Issue by S1SYPHOS** [rosell-dk/webp-convert#1]
Hey there,
quick suggestion: would you publish this for composer (basically creating `composer.json`)?

**Comment by rosell-dk** [comment]
Sure.
- And thanks for the Pull Requests. But how come you closed them again?

**Comment by S1SYPHOS** [comment]
Hey there,
I had a problem with the library I was working on, and it seemed you are  unavailable. So I made further changes and closed the PR.

I‘ll happily commit my changes again!
Cheers

**Comment by rosell-dk** [comment]
Please do :) - I'm not entirely unavailable, though a bit occupied. 

Oh, and there you did. Thanks :)

**Comment by rosell-dk** [comment]
I'll have a look at it tomorrow

**Comment by S1SYPHOS** [comment]
Update:
Would you consider giving me write access? Doesn‘t hurt asking, I guess

**Comment by rosell-dk** [comment]
Sure. Don't know how, though, but I'll figure it out (tomorrow)

**Comment by rosell-dk** [comment]
I have now invited you as a collaborator.

**Comment by rosell-dk** [comment]
An undefined constant "DS" got introduced. I replaced it with "/". But yes, we should update the code to generally use the PHP constant DIRECTORY_SEPARATOR instead of "/"

**Comment by rosell-dk** [comment]
S1SYPHOS - As you are now a collaborator, you might like to know a bit of my plans and motivations regarding this plugin. 

The motivation that drives it is primarily the plan to maintain a Wordpress plugin, which is largely powered by this project. I'm also using the plugin for various websites, but as these a just single websites on servers that I choose, it doesn't require too much of this plugin.

The Wordpress plugin was nearly finished before the summer vacation, but I did not publish it yet in the Wordpress repository . However, since I came back from vaccation, I have had customers pushing on projects, which has taken all my time. So I got sidetracked. I however still hope to get on with the Wordpress plugin. When I do, I will doubtless return to this project as well. The Wordpress plugin is, btw, also here on github: https://github.com/rosell-dk/webp-express
